src/test/java/systemtests/AddressBookSystemTest.java

```diff
@@ -277,7 +275,7 @@ private void assertApplicationStartingStateIsCorrect() {
         assertEquals("", getCommandBox().getInput());
         assertEquals("", getResultDisplay().getText());
         assertListMatching(getPersonListPanel(), getModel().getFilteredPersonList());
-        assertEquals(MainApp.class.getResource(FXML_FILE_FOLDER + DEFAULT_PAGE), getBrowserPanel().getLoadedUrl());
+        assertEquals(getClass().getResource(HelpWindow.SHORT_HELP_FILE_PATH), getBrowserPanel().getLoadedUrl());
```

Comment:
Not sure if it is bad, but I removed it here because I was changing the default page to display a summarized help, but the file directory is different so I don't think I can use MainApp.